doc/api/cli.md

```diff
@@ -144,6 +144,8 @@ Useful when activating the inspector by sending the `SIGUSR1` signal.
 
 Default host is `127.0.0.1`.
 
+See the [security warning](#inspector_security) below.
```

Comment:
``security warning about providing a different `host` ``, or something similar to the added text in the other file? That would give people an indication about the circumstances in which there are security concerns upfront